src/coreComponents/constitutive/unitTests/testCeramicDamage.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 Total, S.A
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+// Source includes
+#include "constitutive/ConstitutiveManager.hpp"
+#include "constitutive/solid/CeramicDamage.hpp"
+
+// TPL includes
+#include <gtest/gtest.h>
+#include <conduit.hpp>
+
+using namespace geos;
+using namespace ::geos::constitutive;
+
+TEST( CeramicDamageTests, testThermalExpansionAndTemperature )
```

Comment:
I expected this to test some of the kernel functions. Is there a plan for that perhaps?